EDMarketConnector.py

```diff
@@ -16,7 +16,7 @@
 from os import chdir, environ
 from os.path import dirname, join
 from time import localtime, strftime, time
-from typing import TYPE_CHECKING, Literal, Optional, Tuple, Union
+from typing import TYPE_CHECKING, Any, Dict, Literal, Optional, Tuple, Union
```

Comment:
if possible, swapping all `Dict` to `dict` could be nice, though note that this breaks 3.8 for those that wanted to use it

Reply:
I've been slinging `A | B` around recently, so we're actually Python 3.10 minimum now.  So, yeah, I'll try to remember to avoid typing.Dict and typing.List etc.